tests/Checker/Preparations/Use_Minimal_Theme_Preparation_Tests.php

```diff
@@ -0,0 +1,61 @@
+<?php
+/**
+ * Tests for the Check_Context class.
+ *
+ * @package plugin-check
+ */
+
+use WordPress\Plugin_Check\Checker\Preparation;
+use WordPress\Plugin_Check\Checker\Preparations\Use_Minimal_Theme_Preparation;
+
+class Use_Minimal_Theme_Preparation_Tests extends WP_UnitTestCase {
+	protected $theme_name;
+	protected $theme_slug;
+	protected $theme_dir;
+
+	public function set_up() {
+		parent::set_up();
+
+		$this->theme_name = 'WP Empty Theme';
+		$this->theme_slug = 'wp-empty-theme';
+		$this->theme_dir  = WP_PLUGIN_DIR . '/' . basename( TESTS_PLUGIN_DIR ) . '/test-content/themes';
```

Comment:
Nit-pick, wouldn't the following result in the same but simpler?
```suggestion
		$this->theme_dir  = TESTS_PLUGIN_DIR . '/test-content/themes';
```